waspc/src/Wasp/Generator/DbGenerator.hs

```diff
@@ -156,12 +115,8 @@ genMigrationsDir spec =
   where
     genProjectMigrationsDir = dbRootDirInProjectRootDir </> dbMigrationsDirInDbRootDir
 
-writeDbSchemaChecksumToFile :: Path' Abs (Dir ProjectRootDir) -> IO ()
```

Comment:
Moved into `src/Wasp/Generator/DbGenerator/Operations.hs` for cyclical deps and for a better fit as to what it is doing/using.